### PR TITLE
Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by HenriqueAJNB**
I've added all the necessary changes to the Brazilian Portuguese translation. 
When this PR is approved, it will resolve issue #43.

The only mention here is that I've followed the instructions in `README.md` file:

> Determine the 2-letter language code (ISO 639-1). See the existing files in git-cheatsheet/lang. 

But it doesn't make any distinction between Brazilian Portuguese and Portuguese from Portugal. It should be `pt-br` instead. 

It's about @ndp or the other maintainer's decision.

### CODE REVIEW

File: git-cheatsheet/lang/pt.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      "cmd": "stash pop",
+      "docs": "Traz de volta as modificações da área temporária (stash) para a máquina local (workspace). O padrão é trazer a última modificação salva. A modificação é removida da área temporária"
+    },
+    "stash list": {
+      "cmd": "stash list",
+      "docs": "Lista as modificações salvas na área temporária (stash)."
+    },
+    "stash show": {
+      "cmd": "stash show [<stash>]",
+      "docs": "Show the changes recorded in the stash as a diff between the stashed state and its original parent. When no <stash> is given, shows the latest one."
```
Comment (ndp) [inline git-cheatsheet/lang/pt.json:166]: Are this and the next item intentionally not translated?

**Comment by ndp** [comment]
A question about `pt` vs. `pt-br`. I don't understand the nuance here, so I need your guidance.

In the U.S., we learn that they are the same "language", but different in pronunciation and a limited amount of vocabulary-- but basically understandable. I am familiar with Spanish, and the differences, and was thinking of the Portuguese in the same way: although there are differences, this particular site will be usable by people from any of the numerous countries that speak Spanish.

But: 
- Is having this Brazilian translation listed as `pt` going to feel wrong or somehow offensive to someone from Portugal or one of the many other countries that speak Portuguese? 
- Do people who speak them think about them as distinct languages? 
- This is a pretty tech-heavy page; how much do the differences between the two languages come into play?

For simplicity, I'd hope there could be just one translation, but I don't want the site to feel wrong.

**Comment by HenriqueAJNB** [comment]
Maybe the website could just show one general main translation as `pt`, it won't be a problem neither offensive to anyone. It's the same situation for Spanish from Mexico and Spanish from Spain. 

What about the `README.md`? Should we keep the `Brazilian Portuguese` or just follow the simplicity with `Portuguese`? Again, there are no such relevant difference between the languages, just a matter of explicitly.

**Comment by HenriqueAJNB** [comment]
> Is having this Brazilian translation listed as pt going to feel wrong or somehow offensive to someone from Portugal or one of the many other countries that speak Portuguese?

No, not at all. It's the same language with a small difference in accent and some misleading words.

> Do people who speak them think about them as distinct languages?

I've seen that there are `pt` and `pt-br`. But I don't know if they are treated like different languages. Just variations of the same language, just like English from the U.S. and from the U.K.

> This is a pretty tech-heavy page; how much do the differences between the two languages come into play?

We can keep only one general `pt`. It will be understandable for every Portuguese speaker.

If you want to learn a little bit more, here is [this article from Babel](https://www.babbel.com/en/magazine/difference-between-brazilian-and-european-portuguese) explaining the differences.

My opinion: let's follow your suggestion to keep `pt` for simplicity.

File: git-cheatsheet/lang/pt.json (hunk 1 above)
Comment (HenriqueAJNB) [reply git-cheatsheet/lang/pt.json:166]: I've forgotten this one. Will commit covering this =)

**Comment by ndp** [comment]
Thank you for your clarification. I'll get this out in a few minutes.

**Comment by waldyrious** [comment]
> This is a pretty tech-heavy page; how much do the differences between the two languages come into play?

Just to add another perspective (I speak Portuguese from Portugal): indeed the two variants are generally intelligible, but the point above is well raised — in tech, terminology tends to differ a bit more markedly than in informal conversation.

That said, this is mostly avoided in this translation because @HenriqueAJNB chose not to translate words like "commit", "branch", "merge", etc. — which is a common practice in Portugal as well, and he added the English term in parenthesis when the meaning may not be obvious (e.g. fila/index).

AFAICT, the only striking difference here is the translation of "file", which here in Portugal we translate as "ficheiro", whereas in Brazil it's "arquivo". Still, it's likely that people in tech circles will be aware of the Brazilian variant, so I suppose it will be more of an inconvenience than a true barrier for understanding.

Maybe in the future some of the wording could be adjusted in ways that would sound OK in both variants (e.g. "inclusas" vs "incluídas", "deletar" vs. "apagar" or "eliminar", etc.), but for the time being this is perfectly serviceable (barring a couple typos and some untranslated strings). Nice work, @HenriqueAJNB!

**Pull request merged**